docs/android-frontend-testing.md

```diff
@@ -142,4 +142,4 @@ You can then vary the device (e.g. the locale or language), or by using a small
 
 ## Conclusion
 
-Much of these will be ameliorated by testing tooling. We appreciate that the current situation is less than ideal.
\ No newline at end of file
+Much of these will be ameliorated by testing tooling. We appreciate that the current situation is less than ideal.
```

Comment:
Can't see what the difference is here, even after a good squint 🤨

Reply:
Sorry. I think this was just kvetching that there was no newline at the end of the file. 🤷 
I should probably just disable that rule in my local mdlint config file.